sbt-dumpschema-plugin/src/main/scala/org/virtuslab/akkasaferserializer/DumpSchemaPlugin.scala

```diff
@@ -0,0 +1,24 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+
+class DumpSchemaPlugin(override val global: Global) extends Plugin {
+  override val name: String = "dump-schema-plugin"
+  override val description: String = ""
+
+  private val pluginOptions = new DumpSchemaOptions("/tmp")
```

Comment:
Sooo by default the output file would be placed directly under `/tmp`?

Reply:
Maybe some uniquely named temp dir will be better? or does it even matter at all?

Reply:
This is a placeholder argument. If no directory is specified in compiler plugin arguments, then the compiler plugin disables itself.

Reply:
Ok pls mention that in a comment then